docs/tasks/configure-pod-container/configure-persistent-volume-storage.md

```diff
@@ -0,0 +1,196 @@
+---
+redirect_from:
+- "/docs/user-guide/persistent-volumes/walkthrough/"
+- "/docs/user-guide/persistent-volumes/walkthrough.html"
+title: Configuring a Pod to Use a PersistentVolume for Storage
+---
+
+{% capture overview %}
+
+This page shows how to configure a Pod to use a PersistentVolume for stoarage.
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+You need to have a Kubernetes cluster that has only one Node, and the kubectl
+command-line tool must be configured to communicate with your cluster. If you
+do not already have a single-node cluster, you can create one by using
+[Minikube](/docs/getting-started-guides/minikube).
+
+{% endcapture %}
+
+{% capture steps %}
+
+## Creating an index.html file on your Node
+
+Open a shell to the Node in your cluster. How you open a shell depends on how
+you set up your cluster. For example, if you are using Minikube, you can open a
+shell to your Node by entering `minikube ssh`.
+
+In your shell, create a `/tmp/data` directory:
+
+    mkdir /tmp/data
+
+In the `/tmp/data` directory, create an `index.html` file:
+
+    echo 'Hello from Kubernetes storage' > /tmp/data/index.html
+
+## Creating a PersistentVolume
+
+In this exercise, you create a *hostPath* PersistentVolume. Kubernetes supports
+hostPath for development and testing on a single-node cluster. A hostPath
```

Comment:
> uses a file or directory on the Node that is running the Pod.

WDYT of

uses a file or directory on the Node to emulate network attached storage.